src/main/scala/zio/sqs/producer/Producer.scala

```diff
@@ -174,29 +170,23 @@ object Producer {
   private[sqs] def buildSendMessageBatchRequest[T](queueUrl: String, serializer: Serializer[T])(entries: List[SqsRequestEntry[T]]): SqsRequest[T] = {
     val reqEntries = entries.zipWithIndex.map {
       case (e: SqsRequestEntry[T], index: Int) =>
-        SendMessageBatchRequestEntry
-          .builder()
-          .id(index.toString)
-          .messageBody(serializer(e.event.data))
-          .messageAttributes(e.event.attributes.asJava)
-          .messageGroupId(e.event.groupId.orNull)
-          .messageDeduplicationId(e.event.deduplicationId.orNull)
-          .delaySeconds(e.event.delay.map(_.getSeconds).getOrElse(0L).toInt)
-          .build()
+        SendMessageBatchRequestEntry(
+          index.toString,
+          serializer(e.event.data),
+          e.event.delay.map(_.getSeconds.toInt),
+          Some(e.event.attributes),
+          None,
+          e.event.deduplicationId,
+          e.event.groupId
+        )
```

Comment:
is it worth to have a named parameters here?
like:
```scala
        SendMessageBatchRequestEntry(
          id = index.toString,
          messageBody = serializer(e.event.data),
...
```
just to improve the readability?

since in the prev version it was more readable with the builder pattern, that is removed now, and looking at 
```scala
SendMessageBatchRequestEntry(
...
None,
...
```
for example, it is not entirely clear what it referes to

and the same comment regarding the construction of `ReceiveMessageRequest` above.

Reply:
Definitely, I was just working through the changes as quickly as possible to get things to compile and test. Now is the time for cleaning up.